assets/home/components/HomeApp.tsx

```diff
@@ -32,45 +30,13 @@ const modals: any = {
     personalizeHome: PersonalizeHomeSettingsModal,
 };
 
-interface IState {
-    activeOptionId: 'default' | 'my-home';
-}
-
-interface IStateProps {
-    cards: Array<any>;
-    personalizedDashboards: Array<IPersonalizedDashboardsWithData>;
-    itemsByCard: any;
-    products: Array<any>;
-    user: string;
-    userType: string;
-    userSections: any;
-    company: string;
-    itemToOpen: any;
-    modal: any;
-    activeCard: string;
-    previewConfig: any;
-    listConfig: any;
-    detailsConfig: any;
-    topics: Array<ITopic>;
-    isSearchEnabled: boolean;
-    filterGroupLabels: any;
-    currentUser: any;
-}
+type IProps = IStateProps & IDispatchProps;
 
-interface IDispatchProps {
-    openItemDetails: (item: any, state: any) => void;
-    fetchItems: () => any;
-    personalizeHome: () => void;
-    actions: Array<IItemAction>;
-    fetchCardExternalItems: (cardId: string, cardLabel: string) => any;
-    fetchCompanyCardItems: () => any;
-    followStory: (item: any) => void;
-    downloadMedia: (href: any, id: any) => any;
+interface IOwnProps {
```

Comment:
`IOwnProps` is a bit misleading here for 2 reasons:
* Typically `ownProps` naming convention is used with Redux (for `map[X]ToProps` functions, props passed in from parent component vs props passed in from Redux state)
* This interface is defining internal state to the component and not a property that is passed in

Reply:
right fixed that